script/readyhelper.js

```diff
@@ -0,0 +1,83 @@
+define(
+  'bigscreenplayer/readyhelper', [
+    'bigscreenplayer/models/mediastate',
+    'bigscreenplayer/models/windowtypes',
+    'bigscreenplayer/models/livesupport'
+  ],
+  function (MediaState, WindowTypes, LiveSupport) {
+    var ReadyHelper = function (initialPlaybackTime, windowType, liveSupport, callback) {
```

Comment:
var ReadyHelper = function is the same as function ReadyHelper. We should keep our code consistent.

Reply:
This style is used elsewhere, which is why I used it in the first place! See for example https://github.com/bbc/bigscreen-player/blob/master/script/playercomponent.js#L17